README.md

```diff
@@ -2,7 +2,7 @@
   <img src="https://forthebadge.com/images/badges/made-with-python.svg">
 </p>
 
-# MapBot
+# MapBot  [![Build Status](https://travis-ci.com/vishakha-lall/MapBot.svg?branch=gssoc-master)](https://travis-ci.com/vishakha-lall/MapBot)
```

Comment:
@chttrjeankr Great addition here 👍

Reply:
Thought it'd make the `README.md` more attractive.